Update README.md

pdksu opened the pull request:
Doc changes to explain what to do with the USB version of this instrument.

pdksu committed:
Update README.md

Doc changes to explain what to do with the USB version of this instrument.

bicarlsen commented on README.md:
```diff
@@ -1,5 +1,5 @@
 # Oriel Cornerstone 260
-A library allowing easy control over Oriel Cornerstone 260 monochromators.
+A library allowing easy control over an Oriel Cornerstone 260 monochromators with an RS-232C port.
```
Change `monochromators` to `monochromator`, or remove `an`

bicarlsen reviewed (changes requested):
Correct grammar in description.

pdksu committed:
Update README.md

typo fix

pdksu committed:
Update README.md

more details on the USB implementation. Because these rely on proprietary code, I don't think a public repo can go much further.

bicarlsen reviewed (approved):
If possible could you also explain where the `CornerstoneDll` resource comes from?
